lib/compiler/test_runner.zig

```diff
@@ -22,6 +22,11 @@ const crippled = switch (builtin.zig_backend) {
 pub fn main() void {
     @disableInstrumentation();
 
+    if (builtin.zig_backend == .stage2_spirv64) {
```

Comment:
```suggestion
    if (builtin.target.cpu.arch.isSpirV()) {
```

Strictly speaking, this has nothing to do with the backend.